aiida/engine/runners.py

```diff
@@ -235,6 +235,9 @@ def _run(self, process: TYPE_RUN_PROCESS, *args: Any, **inputs: Any) -> Tuple[Di
 
             def kill_process(_num, _frame):
                 """Send the kill signal to the process in the current scope."""
+                if process_inited.is_killing:
+                    LOGGER.critical('runner received interrupt, process %s already being killed', process_inited.pid)
```

Comment:
does this need to be critical or would a warning suffice?
warning is anyhow the default log level

Reply:
yeh sounds good